md/two-main-types-of-deployment.md

```diff
@@ -29,6 +30,20 @@ It is fast because the Bonita Portal and the Bonita Engine run on the same JVM a
 **Drawbacks**
 
 * may not be adapted to some architecture constraints
+## Bonita Portal + Bonita Engine running in a docker container
+
+This is a new deployment option available in the 7.10 version.
```

Comment:
```suggestion
This is a new deployment option available starting from 7.10 version.
```

Reply:
Also the docker image was available in other version. It's just that we document it now